### PR TITLE
Add crontab support for deploy and schedule some cleanup commands

### PR DESCRIPTION
**Pull request by xobb1t**
I scheduled with crontab 2 commands: `manage.py cleanup` and `manage.py delete_expired_reports`.

### CODE REVIEW

File: requirements/development.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 -r base.txt
 
 django-kombu==0.9.2
-Fabric==1.2.0
-pycrypto==2.3
+Fabric==1.4.3
+pycrypto==2.6
+ssh==1.7.14
 paramiko==1.7.7.1
```
Comment (yumike) [inline requirements/development.txt]: Please, remove `paramiko`. It was replaced by `ssh`.

File: project/lint/management/commands/delete_expired_reports.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+
```
Comment (yumike) [inline project/lint/management/commands/delete_expired_reports.py]: Only one blank line between each group of imports is required.

**Comment by xobb1t** [comment]
Good catch, @yumike! Fixed.

File: project/lint/management/commands/delete_expired_reports.py (hunk 2 above)
Comment (yumike) [inline project/lint/management/commands/delete_expired_reports.py]: Also, usage of more than two dots in relative imports is considered a bad practice. Just use an absolute import in this case. See http://mail.python.org/pipermail/python-dev/2004-March/043773.html and http://www.python.org/dev/peps/pep-0328/ ("Guido's Decision" section).

Comment (xobb1t) [inline project/lint/management/commands/delete_expired_reports.py]: Oh, I didn't know about that. Fixed!

**Comment by yumike** [comment]
:beers:

**Pull request merged**